apps/omg/lib/omg/state.ex

```diff
@@ -189,19 +195,26 @@ defmodule OMG.State do
 
   Does its on persistence!
   """
+  @decorate measure_event()
```

Comment:
is measuring this event twice, necessary?

Reply:
why twice? I do not understand

Reply:
Actually, it's three times, with little difference 
1. https://github.com/omisego/elixir-omg/pull/616/files/c7d58abfb73bd2f4e76f5c9340320461f24cd446#diff-cfee1fb662913b3948db6872bbce2278R198
2. https://github.com/omisego/elixir-omg/pull/616/files/c7d58abfb73bd2f4e76f5c9340320461f24cd446#diff-cfee1fb662913b3948db6872bbce2278R212
3. https://github.com/omisego/elixir-omg/pull/616/files/c7d58abfb73bd2f4e76f5c9340320461f24cd446#diff-731c57d623f5cd277dc2fcaf9abcae43R33

Reply:
ok, 
1. whole time to form_block
2. multi_update (only time DB update use by multiple functions)
3. more tricky multi_update with waiting for a response (waiting to get into the gen server with long message queues, can be problematic. can tell us about the change of approach sending messages)

Reply:
2 and 3 are the same